package.json

```diff
@@ -20,6 +20,7 @@
     "traverse": "^0.6.6"
   },
   "devDependencies": {
+    "clayhandlebars": "^5.0.1",
```

Comment:
hmmm why do we require `clayhandlebars` here? 🤔

Reply:
#83 moved `clayhandlebars` to the package's `peerDependencies`, but our unit tests explicitly depend on it:

https://github.com/clay/amphora-html/blob/5a55ab02c9e9c68e6a66479f6a50e952efc11181/lib/setup.test.js#L7

My expectation is that adding it to `devDependencies` does not make it available to downstream packages---it's still a peer dependency as far as they're concerned.